### PR TITLE
Adds support for passing config and assessment plan URL for CI

### PR DESCRIPTION
**Pull request by credo-nate**
## Describe your changes

Enabled these environment variables for use in CI jobs.

`CREDOAI_LENS_CONFIG_JSON_B64` - Base64 encoded json

Example:
```
export CREDOAI_LENS_CONFIG_JSON_B64=$(cat <<EOF
{
  "api_key": "secret",
  "tenant": "credoai",
  "api_server": "https://api.credo.ai"
}
EOF
)
```

`CREDOAI_LENS_PLAN_URL` - regular string

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have built basic tests for new functionality (particularly new evaluators)
- [ ] If new libraries have been added, I have checked that [readthedocs](https://readthedocs.org/projects/credoai-lens/) API documentation is constructed correctly
- [ ] Will this be part of a major product update? If yes, please write one phrase about this update.

## Extra-mile Checklist
- [ ] I have thought expansively about edge cases and written tests for them

### CODE REVIEW

**Comment by credo-nate** [comment]
Error right now is 

```
FAILED tests/test_integration.py::test_integration - AttributeError: 'str' object has no attribute 'removesuffix'
```

where removesuffix was introduce in Python 3.9 but CredoAI Lens is supposed to support 3.9+.  @esherman-credo

**Comment by esherman-credo** [comment]
Per the [environment.yml](https://github.com/credo-ai/credoai_lens/blob/develop/environment.yml), we only enforce Python >= 3.8 right now.

Upping that to 3.9 probably wouldn't be a bad idea.

In the near term, [this](https://stackoverflow.com/questions/1038824/how-do-i-remove-a-substring-from-the-end-of-a-string) would suffice.

**Review by esherman-credo — CHANGES_REQUESTED** [review]
Waiting on confirmation re: using `endswith` rather than `removesuffix`

File: tests/conftest.py [hunk 1]
```diff
@@ -402,11 +406,17 @@ def frozen_validation_data():
 
 @fixture(scope="session")
 def config_path_in():
-    return  # TODO!!! FILL IN WITH GITHUB SECRET FOR CONFIG FILE (API TOKEN)
-    # return ""
+    return os.getenv("CREDOAI_LENS_CONFIG_PATH", None)
```
[the remark this answers is not in the document]
Comment (credo-nate) [reply tests/conftest.py:409]: Not necessarily.  If the environment variable is missing, it will look for the config in the normal place.

**Review by esherman-credo — APPROVED** [review]
